src/test/rustdoc-json/reexport/glob_extern.rs

```diff
@@ -0,0 +1,17 @@
+// edition:2018
+
+#![no_core]
+#![feature(no_core)]
+mod mod1 {
+    extern "C" {
+        pub fn public_fn();
+        fn private_fn();
+    }
+}
+
+pub use mod1::*;
+
+// @!has glob_extern.json "$.index[*][?(@.name=='mod1')]"
```

Comment:
nit: Personally, would put 2 of these above mod1, and the next 2 above the pub use, makes it clearer which are referred to. Similar comment on some of the files. It would be nice if at least all the files were consistent.

Reply:
Done and force pushed